config-example.yaml

```diff
@@ -30,7 +30,8 @@ keep_alive_monitor:
 # on your farm performance at the specified time of the day.
 daily_stats:
   enable: true
-  time_of_day: 21
+  notify_hour: 21
+  notify_minute: 0
```

Comment:
Thanks for the proposed change.

I think this won't justify breaking backwards compatibility for all existing configs however. (We don't have auto-config migration). I'd be more willing to merge if the implementation supports old style + maybe new style with parsing of the minutes from the time_of_day string which might either be `21` or `21:00`. 

What do you think?

Reply:
Also I suspect the main use-case here is to quickly check notifications are running. We've previously discussed it will be a good idea to have a config flag that will send a daily notification 1 minute after startup: https://github.com/martomi/chiadog/pull/176#discussion_r638080060

Reply:
There's a branch in the works for this, but I haven't had time to properly finish it yet:
https://github.com/martomi/chiadog/tree/feature/startup-switch

Reply:
@martomi thanks for the feedback, I made a quick change so that it doesn't break the backward compatibility of the configs. For me, it's more of providing the flexibility in setting the time :)